src/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1632,6 +1632,16 @@ elementFormDefault="qualified">
         <xs:documentation><!-- _locID_text="PackageTargetFallback" _locComment="" -->Allows packages using alternative monikers to be referenced in this project, which include older (e.g. dnxcore50, dotnet5.x) and Portable Class Library names.</xs:documentation>
       </xs:annotation>
     </xs:element>
+    <xs:element name="AssetTargetFallback" substitutionGroup="msb:Property">
```

Comment:
do these need this? I see it in others but not all of them and I don't know XSD well

```suggestion
    <xs:element name="AssetTargetFallback" type="msb:StringPropertyType" substitutionGroup="msb:Property">
```